packages/flutter/lib/src/material/text_selection_theme.dart

```diff
@@ -16,6 +17,11 @@ import 'theme.dart';
 /// Use [TextSelectionTheme.of] to access the closest ancestor
 /// [TextSelectionTheme] of the current [BuildContext].
 ///
+/// Note, on iOS [TextField] and [SelectableText] cannot access
```

Comment:
Flutter's style guide says not to use things like "note": https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose